views/components/hero.njk

```diff
@@ -28,31 +28,3 @@
         </div>
     </section>
 {% endmacro %}
-
-{% macro superHero(title, imageDefault, imageCandidates) %}
```

Comment:
Inlined the markup for the two versions of this (old and new pages) as this is unique to the homepage.